install.md

```diff
@@ -0,0 +1,6 @@
+# Building
+go tidy
```

Comment:
do we need `go tidy`? doesn't look like a valid command, there is `go mod tidy`, but I wouldn't recommend to run it as a part of the build.